docs/guides/securing-apps/token-exchange.adoc

```diff
@@ -6,6 +6,322 @@ title="Using token exchange"
+    This behaviour is per https://datatracker.ietf.org/doc/html/rfc8693#section-2.2.1[the token exchange specification].
+
+refresh_token::
+    The refresh token. It is included just if `requested_token_type=urn:ietf:params:oauth:token-type:refresh_token` is used and the client has enabled issuing refresh tokens from the token exchange
+
+issued_token_type::
+    The issued requested token type. Same value as `requested_token_type` used in the request.
+
+token_type::
+    Usually `Bearer` if issued token type was access token or refresh token. In case of ID token requested, the value is `N_A`
+
+[[_standard-token-exchange-scope]]
+=== Scopes and audiences
+
+Parameter `scope` in the token exchange request has same meaning like for any other grants. It is optional parameter. When it is not used, the effective client scopes used in the request will be
+the link:{adminguide_link}#_client_scopes[Default client scopes]  of the `requester-client`. When it is used, the effective client scopes will be default scopes together with
+the link:{adminguide_link}#_client_scopes[Optional client scopes]
+
+By default, the used client scopes will add the audiences to the `aud` claim of the token based on the used client scopes and client roles as specified in the link:{adminguide_link}#audience-support[Audience documentation].
+
+The `audience` parameter can be used for filtering of audiences, so that the `aud` claim will contain only the audiences specified by the `audience` parameter. Similarly the client roles in the token will
+be filtered and token will have only client roles of the clients specified by the `audience` parameter.
+
+Moreover, the `audience` parameter can be used to possibly filter client scopes as well. It works in a bit similar way to link:{adminguide_link}#client-scopes-permissions[Client scope permission for users].
+If the client scope does not contain any client roles (EG. does not contain any roles or contains only realm roles), there is no additional filtering of client scopes. However if client scope contains any
+client role mappings, it must include some client roles of the clients requested by the `audience` parameter. Composite roles are taken into the consideration. If the client scope does not contain any client
+roles of the clients requested by the `audience`, the client scope will be filtered.
+
+NOTE: The `audience` parameter can be used to filter the audiences, which are coming from used client scopes. But this parameter will not add additional audiences. When audience parameter is not used, there is
+no filtering. This means that parameter `audience` can be effectively used for "downscoping" the token to make sure that it contain just the requested audiences. On the other hand `scope` parameter is used
+to add optional client scopes and hence it can be used for "upscoping" and adding more scopes.
```

Comment:
```suggestion
no filtering occurs. As a result, the `audience` parameter is effectively used for "downscoping" the token to make sure that it contains only the requested audiences. However, the `scope` parameter is used
to add optional client scopes and hence it can be used for "upscoping" and adding more scopes.
```